src/styles/blocks/footer.scss

```diff
@@ -0,0 +1,179 @@
+.footer {
+  padding: 72px 0 80px;
+  background-color: $color-darkgray;
+  border-top-left-radius: 30px;
+  border-top-right-radius: 30px;
+
+  &__form {
+    margin: 0 0 80px;
+    max-width: 460px;
+
+    @include onTablet {
+      margin: 0 0 120px;
+    }
+  }
+
+  &__content {
+    display: block;
+    margin-bottom: 81px;
+
+    @include onTablet {
+      margin-bottom: 144px;
+    }
+
+    @include onDesktop {
+      margin-bottom: 120px;
+      display: flex;
+      gap: 226px;
+    }
+  }
+
+  &__title {
+    display: flex;
+    align-items: center;
+    margin: 0 0 48px;
+    font-family: "Poppins", sans-serif;
+    font-weight: 600;
+    font-size: 32px;
+    line-height: 150%;
+    color: $color-white;
+
+    @include onTablet {
+      margin: 0 0 56px;
+    }
+  }
+
+  &__form-field {
+    margin: 0 0 40px;
+
+    @include onTablet {
+      margin: 0 0 48px;
+    }
+  }
+
+  &__contact-section {
+    margin: 0 0 40px;
+
+    @include onTablet {
+      margin: 0 0 48px;
+    }
+  }
+
+  &__subtitle {
+    margin: 0 0 8px;
+    font-family: "Poppins", sans-serif;
+    font-weight: bold;
+    font-size: 13px;
+    line-height: 19px;
+    letter-spacing: 3px;
+    text-transform: uppercase;
+    color: $color-white;
+    opacity: 0.5;
+  }
+
+  &__contact-content {
+    margin: 0;
+    font-family: "Open Sans", sans-serif;
+    font-weight: 600;
+    font-size: 24px;
+    line-height: 150%;
+    color: $color-white;
+    transition: color $transition-duration;
+
+    &:hover {
+      color: $color-blue;
+    }
+  }
+
+  &__contact-socials {
+    display: flex;
+    align-items: center;
+    gap: 45px;
+  }
+
+  &__contact-image {
+    display: block;
+    background-repeat: no-repeat;
+    background-position: center;
+    background-size: contain;
+    width: 32px;
+    height: 32px;
+    transition: all $transition-duration;
```

Comment:
avoid using transition for `all`

Reply:
I corrected, but why I don't write `all` ? If I want to use all properties when I hover. I don't understand this.